fundraising/banner.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Banner configuration
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen <kai.nissen@wikimedia.de>
+ */
+( function( Banner ) {
+	'use strict';
+	
+	Banner.config = {
+		tracking: {
+			campaign: 'default',
+			keyword: 'default',
+			accessCode: 'default',
+			baseUrl: 'https://spenden.wikimedia.de/',
+			libUrl: 'https://spenden.wikimedia.de/piwik/piwik.js',
+			trackerUrl: 'https://spenden.wikimedia.de/piwik/piwik.php',
```

Comment:
You're right, this is and the site ID is needed when calling Piwik.getTracker(). Will change that.